win32/Configure.py

```diff
@@ -651,13 +653,16 @@ def doconfig():
         if os.path.exists(os.path.join(openssl_lib, "libeay32.lib")):
             openssl_lib_name = "libeay32.lib"
             openssl_lib_dll = "bin\\libeay32.dll"
+        if os.path.exists(os.path.join(openssl_lib, "libeaycompat32.lib")):
```

Comment:
Ok, if it is for 1.0, then it should be changed from "if" to "elif". This is so that we have the version detection order:

* "libeay32.lib" => OpenSSL 1.1
* "libeaycompat32.lib" => OpenSSL 1.0 FIPS
* "libcrypto.lib" => OpenSSL 1.0